data/hai/hai reveal 1 intro.txt

```diff
@@ -64,17 +854,21 @@ mission "Hai Leaks Response 3"
 	source "Hai-home"
 	destination "Allhome"
 	to offer
-		has "Hai Leaks Response 2: done"
+		or
+			has "Hai Leaks Response 2: done"
+			has "Hai Leaks Response 2: aborted"
 	on offer
 		conversation
-			`You are directed to an official landing pad on your return to Hai-home, and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems excited to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
-			`	A functionary leads you to what appears to be a situation room, filled with monitors, holoprojectors, and other equipment. Everyone takes their seats at a semicircle-shaped table, and the functionary motions you to the middle seat. A massive holoprojector in front of the table turns on.`
+			`You are directed to an official landing pad on your return to Hai-home.`
+			`	A functionary leads you to what appears to be a situation room, filled with monitors, holoprojectors, and other equipment. All of the diplomats are here except for Remington, who arrives moments behind you. Sayari gestures for you to take a seat as she switches on a massive holoprojector embedded in the table.`
+			`	She nods to you and says, "Other plans are proceeding but I asked for you to come back and join the discussion for this. It seems the increase in pirate activity is only worsening over time since the media retraction, and while innocents may be keeping away, that helps us very little."`
```

Comment:
```suggestion
			`You are directed to an official landing pad on your return to Hai-home, and after securing the <ship>, a functionary leads you to what appears to be a situation room, filled with monitors, holoprojectors, and other equipment. All of the diplomats are here except for Remington, who arrives moments behind you.`
			`	Sayari gestures for you to take a seat as she switches on a massive holoprojector embedded in the table. She nods to you and says, "Other plans are proceeding, but I asked for you to come back and join the discussion for this. It seems the increase in pirate activity has only worsened over time since the media retraction, and while innocents may be keeping away, that helps us very little."`
```
Seems weird that the landing pad is singled out in its own line.

`Securing the <ship>` could be improved, I wanted something that could bridge the sentences.